scripts/gen_travis.py

```diff
@@ -108,94 +131,91 @@ def __eq__(self, obj):
 )]
 
 
-all_unusuals = (compilers_unusual + compiler_flag_unusuals
+all_unusuals = (compilers_unusual + feature_unusuals
     + configure_flag_unusuals + malloc_conf_unusuals)
 
 
-gcc_multilib_set = False
-
-
 def get_extra_cflags(os, compiler):
     # We get some spurious errors when -Warray-bounds is enabled.
+    if os == WINDOWS:
+        return ['-fcommon'] if compiler != CL.value else []
```

Comment:
I'm a bit lost on the syntax here; maybe separate lines?

Reply:
No problem, will make it more readable